### ISSUE TITLE
Wwise plugins support - add the possibility to build a Wwise Effect plugin

### ISSUE THREAD

**Issue by martinbussyparis** [enzienaudio/heavy#85]
Wwise Source plugins allow synthesis, analysis of the audio input, etc., but can't affect any audio signal coming from Wwise; Wwise Effect plugins are designed for this and it would be nice to be able to generate both types of Wwise plugins with Heavy.

Heavy-Wwise Effect plugins would support:
- [receive]
- [receive~]
- [send]: this feature would work as the Wwise Meter and send the parameters to RTPCs
- [send~]

**Comment by diplojocus** [comment]
@martinbussyparis Wwise Effect Plugins are coming, we've got it working just need to roll it out on the site. The way it would work is 
- [receive @hv_param 0 1 0.5] would be your RTPC input to the plugin
- [adc~] audio rate input
- [dac~] audio rate output

It should be possible to send messages out of the plugin but I haven't integrated that into the Wwise wrapper yet.

**Comment by martinbussyparis** [comment]
Yes, [adc~] and [dac~] of course... sorry, this is what I meant.

Good to know it is almost there :)

**Comment by jedsound** [comment]
Joe,
Do you have any ETA on that issue?

**Comment by diplojocus** [comment]
@jedsound we're just putting the finishing touches to the new 2016 release and then we'll look at creating effect plugins.
